server/models/migrations/next/migrate.sql

```diff
@@ -1 +1,643 @@
-/* migration script from the version 1.14.0 to the next */
+    /* migration script from the version 1.14.0 to the next */
+    /*
+    @jeremielodi
+    2020-08-06
+    rename all foreign key constraints
+    */
+
+
+
+-- DROP all foreign keys from a table
+
+-- usage : CALL drop_constraints('account');
+
+DELIMITER $$
+
+DROP PROCEDURE IF EXISTS `drop_constraints`$$
+CREATE   PROCEDURE `drop_constraints`(IN _table_name VARCHAR(100))
+BEGIN
+DECLARE _constraint_name VARCHAR(100);
+
+DECLARE done BOOLEAN;
+DECLARE curs1 CURSOR FOR 
+   SELECT CONSTRAINT_NAME
+		 FROM
+	    INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS
+	   WHERE CONSTRAINT_SCHEMA = DATABASE()
+	   AND TABLE_NAME = _table_name;
+	   
+	   
+DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = TRUE;
+
+OPEN curs1;
+  read_loop: LOOP
+  FETCH curs1 INTO _constraint_name;
+    IF done THEN
+      LEAVE read_loop;
+    END IF;
+        
+    SET @drop_foreign_key =  CONCAT('ALTER TABLE ', _table_name, ' DROP FOREIGN KEY ', _constraint_name);
+    PREPARE drop_query FROM @drop_foreign_key;
+    EXECUTE drop_query;
+
+    END LOOP;
+CLOSE curs1;
+END$$
+DELIMITER ;
+
+
+    SET foreign_key_checks = 0;
+
+    CALL drop_constraints('account');
+    ALTER TABLE `account` ADD CONSTRAINT `account_account_type` FOREIGN KEY (`type_id`) REFERENCES `account_type` (`id`),
+    ADD CONSTRAINT `account_entreprise` FOREIGN KEY (`enterprise_id`) REFERENCES `enterprise` (`id`),
```

Comment:
Is 'account_entreprise' spelled correctly?

Reply:
It was a typo